doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -186,6 +186,10 @@ Matter fork
 nRF IEEE 802.15.4 radio driver
 ------------------------------
 
+* Added:
+
+  * Added Kconfig options to configure default CSMA-CA algorithm parameters (:kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MIN_BE_DEFAULT`, :kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MAX_BE_DEFAULT`, :kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MAX_CSMA_BACKOFFS_DEFAULT`)
```

Comment:
```suggestion
  * The Kconfig options to configure default CSMA-CA algorithm parameters (:kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MIN_BE_DEFAULT`, :kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MAX_BE_DEFAULT`, :kconfig:option:`CONFIG_NRF_802154_CSMA_CA_MAX_CSMA_BACKOFFS_DEFAULT`).
```

Reply:
fixed, (spotted also in this comment https://github.com/nrfconnect/sdk-nrf/pull/22933#discussion_r2163244344 )